src/index.js

```diff
@@ -1,4 +1,3 @@
 var Swarm = require('./swarm')
 
 exports = module.exports = Swarm
-exports.singleton = new Swarm()
```

Comment:
not sure if this is used anywhere but better not since that singleton `Swarm` could not add transports.